tests/pytests/test_info.py

```diff
@@ -52,6 +52,37 @@ def testInfo(env):
 
   #print info
 
+def test_vecsim_hnsw_info():
+  env = Env(protocol=3)
+  dim = 2
+
+  for alg in ["HNSW", "FLAT"]:
+    info_expected = {"identifier": "vec", "attribute": "vec", "type": "VECTOR", "algorithm": alg,
+                     "dim": dim, "flags": []}
+    hnsw_params = {"M": 12, "ef_construction": 100} if alg == "HNSW" else {}
```

Comment:
```suggestion
    additional_params = {"M": 12, "ef_construction": 100} if alg == "HNSW" else {}
```
would be clearer IMO